rm-datasource/src/main/java/io/seata/rm/datasource/exec/InsertExecutor.java

```diff
@@ -43,6 +45,7 @@
  * @date 2019-03-21 21:30:02
  */
 public class InsertExecutor<T, S extends Statement> extends AbstractDMLBaseExecutor<T, S> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(InsertExecutor.class);
```

Comment:
blank one line

Reply:
Added one blank line after the class, and add one blank line after the logger.